src/components/authors/authors.ts

```diff
@@ -0,0 +1,25 @@
+import './authors.scss';
+import WebComponent, { createElementFromString } from '../../common/WebComponent';
+
+
+class AuthorsComponent implements WebComponent {
+  render(): Element {
+    return createElementFromString(`<main>
```

Comment:
`<main>` tag is used to specify the main content of the `<body>`. What if I would like to use this component, but also the other one or if I would like to use this component twice?